collector/aws/elbv2.go

```diff
@@ -89,10 +93,13 @@ func (el *ELBV2Manager) Detect() ([]DetectedELBV2, error) {
 	now := time.Now()
 
 	for _, instance := range instances {
+		var cloudWatchNameSpace string
+		if cloudWatchNS, found := loadBalancerCWNameSpace[*instance.Type]; found {
```

Comment:
what are you expecting to happen when the instance type not found in `loadBalancerCWNameSpace`